proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/ClientSettings.java

```diff
@@ -190,4 +192,21 @@ public void encode(ByteBuf buf, ProtocolUtils.Direction direction, ProtocolVersi
   public boolean handle(MinecraftSessionHandler handler) {
     return handler.handle(this);
   }
+
+  @Override
+  public boolean equals(@Nullable final Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    final ClientSettings that = (ClientSettings) o;
+    return viewDistance == that.viewDistance && chatVisibility == that.chatVisibility && chatColors == that.chatColors && difficulty == that.difficulty && skinParts == that.skinParts && mainHand == that.mainHand && chatFilteringEnabled == that.chatFilteringEnabled && clientListingAllowed == that.clientListingAllowed && Objects.equals(locale, that.locale);
```

Comment:
This code definitely does not follow the established checkstyle rules in Velocity, if you try to compile the project it will give an error

Reply:
fixed